uber/config.py

```diff
@@ -73,9 +73,19 @@ def make_enum(self, enum_name, section, prices=False):
                 lookup[val] = desc
 
         enum_name = enum_name.upper()
-        setattr(self, enum_name + '_OPTS', opts)
-        setattr(self, enum_name + '_VARS', varnames)
-        setattr(self, enum_name + ('' if enum_name.endswith('S') else 'S'), lookup)
+        # When loading plugin's configs, we want to make sure we don't override existing enums
+        if hasattr(self, enum_name + '_OPTS'):
```

Comment:
Oh derp.  Actually we already DO have a separate ini file for Sideboard, so a lot of what I said in my previous comment is wrong anyway!  (This is what I get for trying to chime in on a ticket in the middle of a series of 12-hour workdays and after-work MAGFest stuff, bleh.)